### PR TITLE
[ES] Update `Beatmap Nominators`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Beatmap_Nominators/Expectations/es.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Expectativas para los Beatmap Nominators
+# Expectativas de los Beatmap Nominators
 
-*Véase: [Reglas del Beatmap Nominator](/wiki/People/Beatmap_Nominators/Rules)*
+*Véase también: [Reglas para los Beatmap Nominator](/wiki/People/Beatmap_Nominators/Rules)*
```
Comment (agatemosu) [inline wiki/People/Beatmap_Nominators/Expectations/es.md:3]:
```suggestion
*Véase también: [Reglas para los Beatmap Nominators](/wiki/People/Beatmap_Nominators/Rules)*
```

File: wiki/People/Beatmap_Nominators/es.md [hunk 2]
```diff
@@ -1,47 +1,44 @@
...
   - BN
   - BNG
-outdated_translation: true
-outdated_since: 0b232d41079cbc4141b26a40be57af0eae4a87ab
 ---
 
 # Beatmap Nominators
 
-*Véase también: [Reglas de los Beatmap Nominators](Rules), [Expectativas para los Beatmap Nominators](Expectations), [Información general para los Beatmap Nominators](General_Information), y [Convertirse en Beatmap Nominator](Becoming_a_Beatmap_Nominator)*
+*Véase también: [Reglas para los Beatmap Nominators](Rules), [Expectativas de los Beatmap Nominators](Expectations), [Información general para los Beatmap Nominators](General_Information) y [Convertirse en Beatmap Nominator](Becoming_a_Beatmap_Nominator)*
```
Comment (agatemosu) [inline wiki/People/Beatmap_Nominators/es.md:9]:
```suggestion
*Véase también: [Reglas para los Beatmap Nominators](Rules), [Expectativas de los Beatmap Nominators](Expectations), [Información general de los Beatmap Nominators](General_Information) y [Convertirse en Beatmap Nominator](Becoming_a_Beatmap_Nominator)*
```

File: wiki/People/Beatmap_Nominators/es.md [hunk 3]
```diff
@@ -1,47 +1,44 @@
...
 
-Los nuevos miembros de los Beatmap Nominators empiezan con un periodo largo de prueba de un mes. Si su comportamiento y nominaciones son satisfactorias, serán promovidos a los Beatmap Nominators completos después de una [evaluación](/wiki/People/Nomination_Assessment_Team/Evaluations) positiva. De otra manera, seguirán en el periodo de prueba por otro mes o serán removidos de los Beatmap Nominators.
+Los nuevos miembros de Beatmap Nominators comienzan con un período de prueba de un mes. Si sus nominaciones y conducta son satisfactorios, serán promovidos a Beatmap Nominators completos luego de una [evaluación](/wiki/Personas/Nominación_Assessment_Team/Evaluaciones) positiva. De lo contrario, permanecerán en período de prueba por otro mes o serán eliminados de Beatmap Nominators.
```
Comment (agatemosu) [inline wiki/People/Beatmap_Nominators/es.md:33]:
```suggestion
Los nuevos miembros de Beatmap Nominators comienzan con un período de prueba de un mes. Si sus nominaciones y conducta son satisfactorios, serán promovidos a Beatmap Nominators completos luego de una [evaluación](/wiki/People/Nomination_Assessment_Team/Evaluations) positiva. De lo contrario, permanecerán en período de prueba por otro mes o serán eliminados de Beatmap Nominators.
```

File: wiki/People/Beatmap_Nominators/es.md [hunk 4]
```diff
@@ -1,47 +1,44 @@
...
+Cuando un Beatmap Nominator se pone en período de prueba, no se le puede volver a poner en período de prueba por el mismo motivo durante aproximadamente un año. Por ejemplo, si un Beatmap Nominator es puesto en período de prueba por mala conducta, será eliminado de Beatmap Nominators si exhibe la misma mala conducta nuevamente recientemente, incluso si es un Beatmap Nominator completo durante la segunda infracción.
 
 ## Miembros del equipo
 
 Las páginas [Beatmap Nominators](https://osu.ppy.sh/groups/28) y [Beatmap Nominators (Probationary)](https://osu.ppy.sh/groups/32) listan a todos los miembros del equipo.
 
-Las tablas a continuación indican el(los) modo(s) de juego en donde cada Beatmap Nominator es formalmente calificado para nominar. *Nota: Todos los Beatmap Nominator hablan Inglés en adición al(a los) idioma(s) listado(s) debajo a menos que se especifique otra cosa.*
+Las tablas a continuación indican los modos de juego en donde cada Beatmap Nominator es formalmente calificado para nominar. *Nota: Todos los Beatmap Nominators hablan inglés en adición a a los idiomas listados debajo a menos que se especifique otra cosa.*
```
Comment (UnknownOsu) [inline wiki/People/Beatmap_Nominators/es.md:41]:
```suggestion
Las tablas a continuación indican los modos de juego en donde cada Beatmap Nominator es formalmente calificado para nominar. *Nota: Todos los Beatmap Nominators hablan inglés en adición a los idiomas listados debajo a menos que se especifique otra cosa.*
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Comment by UnknownOsu** [comment]
No me deja solicitar el cambio, pero en el artículo `Beatmap_Nominators/Expectations` en la línea 14 hay una doble `s` en `clasificado`

**Review by TicClick — APPROVED** [review]

**Pull request merged**